.github/workflows/sync_docs.md

```diff
@@ -0,0 +1,72 @@
+Workflow file: [_sync_docs.yaml](_sync_docs.yaml)
+
+## Usage
+Add `.yaml` file to `.github/workflows/`
+```yaml
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: Sync docs from Discourse
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: # Refer to Run schedule below
+  push:
+    branches:
+      - main
+
+jobs:
+  sync-docs:
+    name: Open PR with docs changes
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write  # Needed to login to Discourse
```

Comment:
Any idea about this, @deusebio ?

Reply:
Uhm, not sure, but I don't think so. Anyhow, I believe this is anyway needed to have permission to create commits with the Discourse contents as well as update the tag. 

Interestingly, in the [Kafka operator](https://github.com/canonical/kafka-operator/blob/main/.github/workflows/ci.yaml#L47), the comment seem more appropriate, so it may have been some typos along the way :D